electron/config.ts

```diff
@@ -461,7 +461,8 @@ class GlobalConfigV0 extends GlobalConfig {
       maxWorkers: 0,
       minimizeOnLaunch: false,
       nvidiaPrime: false,
-      otherOptions: '',
```

Comment:
we need some way to port the old config to the new config, this would break games for people already using the `otherOptions` config, maybe we should still show and support reading this (maybe using readonly on the input field to prevent changing it) and with a `deprecated` label telling user to use the new method?

Reply:
i added a mapper in main.ts. Have a look.